PullUpController/PullUpController.swift

```diff
@@ -379,7 +401,8 @@ extension UIViewController {
                                   initialStickyPointOffset: CGFloat,
                                   animated: Bool) {
         assert(!(self is UITableViewController), "It's not possible to attach a PullUpController to a UITableViewController. Check this issue for more information: https://github.com/MarioIannotta/PullUpController/issues/14")
-        addChild(pullUpController)
+//        addChild(pullUpController)
+        addChildViewController(pullUpController)
```

Comment:
could you please revert this change? I want to keep the master branch sync with the latest swift version